integration-tests/scaffolding/extrinsicHelpers.ts

```diff
@@ -0,0 +1,45 @@
+import { ApiRx } from "@polkadot/api";
+import { KeyringPair } from "@polkadot/keyring/types";
+import { Event } from "@polkadot/types/interfaces";
+import { firstValueFrom, filter } from "rxjs";
+import { EventMap, groupEventsByKey, Sr25519Signature } from "./helpers";
+
+type AddKeyData = {
+    msaId?: any;
+    expiration?: any;
+    newPublicKey?: any;
+}
+
+/** Schema Extrinsics */
```

Comment:
If we continue to use helpers like this, we can separate these helpers by pallet.

Reply:
These look great!